apps/infra/src/app/production/shared.ts

```diff
@@ -150,6 +150,10 @@ export const environment = pulumi
       { name: 'DRIVE_DISK', value: 's3' },
       { name: 'ASW_REGION', value: 'eu-central-1' },
       { name: 'S3_BUCKET', value: 'latitude-llm-bucket-production' },
+      {
+        name: 'PUBLIC_S3_BUCKET',
+        value: 'latitude-llm-public-bucket-production',
+      },
```

Comment:
I'd recommend moving the infra changes to another PR and merge that PR beforehand. Also you'll need to manually deploy workers and websockets in order for this envvar to be available there (gateway and web are automatically updated when you deploy)